Sources/PackageDescription/Target.swift

```diff
@@ -253,6 +242,7 @@ public final class Target {
         resources: [Resource]? = nil,
         publicHeadersPath: String?,
         type: TargetType,
+        packageAccess: Bool = false,
```

Comment:
defaulting to false is surprising. most other arguments do not have a default value that impact's behavior so imo this should either not have a default

Reply:
The resources param above for example has a default value. This removes the need to set the value to false explicitly in Target initialization from  static funcs in the earlier versions that do not have this param.

Reply:
I would prefer this does not have a default as it is driving behavior - just like `TargetGroup` did not have a default before

Reply:
that's reasonable; will make the change